compose/snippets/src/main/java/com/example/compose/snippets/navigation3/scenes/ScenesSnippets.kt

```diff
@@ -73,132 +76,129 @@ public class SinglePaneSceneStrategy<T : Any> : SceneStrategy<T> {
 // [END android_compose_navigation3_scenes_2]
 
 // [START android_compose_navigation3_scenes_3]
-// --- TwoPaneScene ---
+// --- ListDetailScene ---
 /**
- * A custom [Scene] that displays two [NavEntry]s side-by-side in a 50/50 split.
+ * A [Scene] that displays a list and a detail [NavEntry] side-by-side in a 40/60 split.
+ *
  */
-class TwoPaneScene<T : Any>(
+class ListDetailScene<T : Any>(
     override val key: Any,
     override val previousEntries: List<NavEntry<T>>,
-    val firstEntry: NavEntry<T>,
-    val secondEntry: NavEntry<T>
+    val listEntry: NavEntry<T>,
+    val detailEntry: NavEntry<T>,
 ) : Scene<T> {
-    override val entries: List<NavEntry<T>> = listOf(firstEntry, secondEntry)
+    override val entries: List<NavEntry<T>> = listOf(listEntry, detailEntry)
     override val content: @Composable (() -> Unit) = {
         Row(modifier = Modifier.fillMaxSize()) {
-            Column(modifier = Modifier.weight(0.5f)) {
-                firstEntry.Content()
+            Column(modifier = Modifier.weight(0.4f)) {
+                listEntry.Content()
             }
-            Column(modifier = Modifier.weight(0.5f)) {
-                secondEntry.Content()
+            Column(modifier = Modifier.weight(0.6f)) {
+                detailEntry.Content()
             }
         }
     }
 
     companion object {
-        internal const val TWO_PANE_KEY = "TwoPane"
+        internal const val LIST_KEY = "ListDetailScene-List"
```

Comment:
I was thinking about metadata ownership and in the case where there's a 1-1 relationship between the strategy and its scene it makes more sense for the strategy to own the metadata and helper functions. 

This is because it simplifies the call site - the caller only needs to import and use the strategy. They don't need to know anything about the scene the strategy creates. 

Please could you move this into the strategy (either in this PR or in a separate PR).